sys/sys/signal.h

```diff
@@ -437,9 +437,9 @@ struct siginfo64 {
 #define	PROT_CHERI_PERM		5	/* Capability permission fault	*/
 #define	PROT_CHERI_IMPRECISE	7	/* Imprecise bounds fault	*/
 #define	PROT_CHERI_STORELOCAL	8	/* Store-local fault		*/
-#define	PROT_CHERI_CCALL	9	/* CCall fault			*/
-#define	PROT_CHERI_CRETURN	10	/* CReturn fault		*/
+#define	PROT_CHERI_CINVOKE	9	/* CInvoke fault		*/
 #define	PROT_CHERI_SYSREG	11	/* Capability system register fault */
+#define	PROT_CHERI_UNALIGNED_BASE 12	/* Unaligned base address.      */
```

Comment:
Adding a new `PROT_CHERI_*` for this is a bit hacky.   None of the others are good fits though.